dev-tools/packer/readme.md.j2

```diff
@@ -0,0 +1,23 @@
+# Welcome to {{.beat_name}} {{.version}}
+
+{{.beat_description}}
+
+## Getting Started
+
+To get running with your beat, setup elasticsearch on your localhost and start it with:
+
+     ./{{.beat_name}}  -c {{.beat_name}}.yml -e
+
+This will start the beat and send to do your elasticsearch instance. To load the dashboards for {{.beat_name}} into Kibana, run:
```

Comment:
I adjusted it to: This will start the beat and send the data to your elasticsearch instance.